modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -278,13 +304,26 @@ export default class NestedTabs extends Base {
 		if ( this.getSettings( 'autoExpand' ) ) {
 			this.activateDefaultTab();
 		}
+
+		this.setHorizontalScrollAlignment();
 	}
 
 	onEditSettingsChange( propertyName, value ) {
 		if ( 'activeItemIndex' === propertyName ) {
 			this.changeActiveTab( value, false );
 		}
 	}
+	onElementChange( propertyName ) {
+		const propsToListen = [
+			'horizontal_scroll',
```

Comment:
@ArmanPurtoian This will not work for responsive controls like `horizontal_scroll`.
rather use something like `0 === propertyName.indexOf( 'horizontal_scroll' )`
Because this needs to check for horizontal_scroll_tablet, horizontal_scroll_mobile etc as well, right?